assets/js/src/core/modules/element/listing/decorators/general-filters/view-layer/components/sidebar/tabs/filters/field-filters/field-filters-container.tsx

```diff
@@ -20,20 +23,24 @@ import { FieldFilters, type FieldFiltersProps } from '@Pimcore/components/field-
 import { useFilter } from '../provider/filter-provider/use-filter'
 
 const FILTER_FIELD_KEY_IGNORE_LIST = ['size']
+const FILTER_TYPE_OVERRIDE_WHITELIST = ['dataobject.adapter', 'dataobject.objectbrick']
```

Comment:
I think it would be good to also move that into the dynamic types of the filters. 

Something like isOverridingFilterType or similiar.
With this we would have the benefit that we still can control the behavior via the plugins.